### ISSUE TITLE
Fix S5332 FN: Support utf-8 string literals

### ISSUE THREAD

**Issue by gregory-paidis-sonarsource** [SonarSource/sonar-dotnet#6297]
### Description

FN for S5332 (ClearTextProtocolsAreSensitive).

Using C#11 utf-8 string literals, the rule is not raising an issue.

### Repro steps

```csharp
var simple = "http://foo.com"u8; // FN
var raw = """http://foo.com"""u8; // FN
var raw_multiline = """
http://foo.com
"""u8; // FN
```

### Expected behavior

The rule should raise an issue.

### Actual behavior

The rule does not raise an issue.
